guides/doc-Planning_Guide/topics/Glossary.adoc

```diff
@@ -128,9 +128,10 @@ In relationship to a host, erratum is *applicable* if it updates a package insta
 
 
 [[varl-Glossary_of_Terms-External_Node_Classifier]]
-*External Node Classifier*:: A Puppet construct that provides additional data for a Puppet server to use when configuring hosts.
+*External Node Classifier*:: A construct that provides additional data for a server to use when configuring hosts.
```

Comment:
Just construct read weird, perhaps a component?

Reply:
@ian your call

Reply:
Comments made on PR#600, the precursor of this PR 
 @ekohl ekohl on Jul 22

I've been trying to convince people that Foreman should adopt the term ENC and not consider it a Puppet-only concept. However, this appears to fall on deaf ears. Maybe at this point I'm just ranting.

melcorr on Jul 23 Member 
And how would that look?
Suggested change
*External Node Classifier*:: A Puppet construct that provides additional data for a Puppet server to use when configuring hosts.
*External Node Classifier*:: A construct that provides additional data for a server to use when configuring hosts.
Like this?

That's why it was changed.